srcpkgs/sublime-text3-bin/template

```diff
@@ -0,0 +1,55 @@
+# Template file for 'sublime-text3-bin'
+pkgname=sublime-text3-bin
+version=3211
+revision=1
+archs="i686 x86_64"
+wrksrc="sublime_text_3"
+hostmakedepends="w3m"
+depends="libpng gtk+ hicolor-icon-theme desktop-file-utils"
+short_desc="Sophisticated text editor for code, markup and prose"
+maintainer="Andrea Brancaleoni <miwaxe@pompel.me>"
+license="custom:EULA"
+homepage="http://www.sublimetext.com/3dev"
+_license_checksum=33929b71625d13dacf2a0a5853171b9c04058f71e2955ee660b8d0f8dda45ed1
+repository="nonfree"
+restricted=yes
+nopie=yes
+replaces="sublime-merge>0"
```

Comment:
this didn't work in my testing, but it's how `sublime-merge{,-bin}` did it. what's the proper way?

Reply:
set `sublime-text3>0` and not `sublime-merge>0`

Reply:
unneeded, I changed approach: https://github.com/void-linux/void-packages/pull/35896#issuecomment-1059963424